install/_lib.sh

```diff
@@ -8,6 +8,47 @@ umask 002
 log_file="sentry_install_log-`date +'%Y-%m-%d_%H-%M-%S'`.txt"
 exec &> >(tee -a "$log_file")
 
+# Sentry SaaS uses stock Yandex ClickHouse, but they don't provide images that
```

Comment:
Is there a specific reason to move the logic from `detect-platform.sh` here? Thought it might be potentially easier to read if the logic existed in another file

Reply:
It is needed below for the `send_event` function, so it had to be moved here.

Reply:
Why do we need to define `send_event` in `_lib.sh` instead of in `error-handling.sh`?

Reply:
Moved to `error-handling.sh`